function/services/ssm/index.test.js

```diff
@@ -0,0 +1,18 @@
+'use strict';
```

Comment:
Test here is fine, but you're still falling below the coverage threshold on this file.